libs/libwebsockets/Makefile

```diff
@@ -97,6 +102,18 @@ ifeq ($(BUILD_VARIANT),full)
     CMAKE_OPTIONS += -DLWS_WITH_CGI=ON
 endif
 
+ifeq ($(BUILD_VARIANT),minimal-server)
+   CMAKE_OPTIONS += -DLWS_WITH_STATIC=OFF
+   CMAKE_OPTIONS += -DLWS_WITH_SHARED=ON
+   CMAKE_OPTIONS += -DLWS_WITH_SSL=OFF
+   CMAKE_OPTIONS += -DLWS_WITH_ZLIB=OFF
+   CMAKE_OPTIONS += -DLWS_WITHOUT_CLIENT=ON
+   CMAKE_OPTIONS += -DLWS_WITHOUT_EXTENSIONS=ON
+   CMAKE_OPTIONS += -DLWS_SSL_CLIENT_USE_OS_CA_CERTS=OFF
```

Comment:
Yes, if you want the most minimal version possible without ssl support. Like in my case :)